examples/globe_wfs_extruded.js

```diff
@@ -0,0 +1,122 @@
+/* global itowns, document, renderer */
+// # Simple Globe viewer
+
+// Define initial camera position
+var positionOnGlobe = { longitude: 4.818, latitude: 45.7354, altitude: 3000 };
+var promises = [];
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
+function addLayerCb(layer) {
+    return globeView.addLayer(layer);
+}
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+itowns.proj4.defs('EPSG:3946',
+    '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+// Add one imagery layer to the scene
+// This layer is defined in a json file but it could be defined as a plain js
+// object. See Layer* for more info.
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb));
+
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+
+function colorFunctionLine(layer, node, featureCollection) {
+    var i;
+    var featureProperties;
+    var rgb;
+    var colors = [];
+
+    for (i = 0; i < featureCollection.features.length; i++) {
+        featureProperties = featureCollection.features[i].properties;
+
+        rgb = featureProperties.couleur.split(' ');
+        colors.push(new itowns.THREE.Color(rgb[0] / 255, rgb[1] / 255, rgb[2] / 255));
+    }
+
+    itowns.FeatureProcessing.assignColorsToFeatureCollection(
+        featureCollection, featureCollection.children[0], colors);
+
+    featureCollection.children[0].material.linewidth = 5;
+}
+
+globeView.addLayer({
+    update: itowns.FeatureProcessing.update(colorFunctionLine),
+    convert: itowns.Feature2Mesh.convertAndExtrude,
+    url: 'https://download.data.grandlyon.com/wfs/rdata?',
+    protocol: 'wfs',
+    version: '2.0.0',
+    id: 'tcl_bus',
+    typeName: 'tcl_sytral.tcllignebus',
+    level: 14,
+    projection: 'EPSG:3946',
+    style: {
+        altitude: function altitude() { return 180; },
```

Comment:
`altitude` property seems out of place in `style`. Maybe this function should be a parameter of `convertAndExtrude` instead.

Reply:
And of course, the same comment is valid for `extrude`